packages/react-router-dev/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # `@react-router/dev`
 
+### Patch Changes
+- Copy `server.watch` config options from `vite.config.(ts|js)` file to child vite server in development mode.
+
```

Comment:
```suggestion
```

Instead of adding to the changelog manually, you should use a changeset. Check the 2nd comment on this PR for instructions